[GTK] fast/mediastream/media-element-current-time.html is failing since added in r256346

philn opened the pull request:
#### 3e994a2e314219315bf6a8beb90f0e86399eeb94
<pre>
[GTK] fast/mediastream/media-element-current-time.html is failing since added in r256346
<a href="https://bugs.webkit.org/show_bug.cgi?id=208120">https://bugs.webkit.org/show_bug.cgi?id=208120</a>
&lt;rdar://problem/86967699&gt;

Reviewed by Eric Carlson.

* LayoutTests/fast/mediastream/media-element-current-time.html: Rely on events and rvFc to ensure
the test behavior is consistent.

Canonical link: <a href="https://commits.webkit.org/253220@main">https://commits.webkit.org/253220@main</a>
</pre>

eric-carlson reviewed (approved):
This is a nice improvement, thanks.

r=me

philn committed:
[GTK] fast/mediastream/media-element-current-time.html is failing since added in r256346
https://bugs.webkit.org/show_bug.cgi?id=208120
<rdar://problem/86967699>

Reviewed by Eric Carlson.

* LayoutTests/fast/mediastream/media-element-current-time.html: Rely on events and rvFc to ensure
the test behavior is consistent.

Canonical link: https://commits.webkit.org/253220@main
